frontend/src/components/SamplesList.tsx

```diff
@@ -123,12 +129,23 @@ export default function SamplesList({
     setRowCount(data?.samplesConnection.edges.length || 0);
   }, [data]);
 
+  const samples = data?.samplesConnection?.edges.map((e) => e.node) as Sample[];
+
+  const popupParamId = useMemo(() => {
+    if (parentWhereVariables && samples && params) {
+      return getSamplePopupParamId(
+        parentWhereVariables,
+        samples,
+        Object.values(params)?.[0]!
+      );
+    }
+    return undefined;
+  }, [parentWhereVariables, params, samples]);
```

Comment:
This handles when the Samples view == Patient Samples view and returns the `cmoPatientId` to be used in the title/export filename. `dmpPatientId` is used if `cmoPatientId` is not found.